_posts/2016-05-17-reimagining-federal-websites-with-the-draft-us-web-design-standards.md

```diff
@@ -0,0 +1,159 @@
+Miller](http://www.iamandrewmiller.com/), a user-interface designer and
+front-end web developer working with Sandia National Labs, decided to
+use the Draft Standards to [reimagine
+cia.gov](https://medium.com/organizer-sandbox/let-s-redesign-the-u-s-government-cia-gov-efe2070c8146#.7x1cex4k0)
+as well as a prototype for a new [mobile and web
+experience](https://medium.com/@iamandrewmiller/where-s-my-tax-refund-redesigning-irs-gov-53fd2dccb3fc#.1nq2jvy47)
+for the Internal Revenue Service (IRS).
+
+He did this as a design exercise for himself, he says, and as a way to
+test out the [Draft
+Standards](https://playbook.cio.gov/designstandards/) in action.
+
+“I wanted to show that this living pattern library can be successfully
+applied to a static, content-based government website,” he wrote in a
+Medium post outlining his approach. “After familiarizing myself with the
+[Draft Standards], I found it easier to iterate and build off of the
+visual standards.”
+
+We reached out to Andrew to ask him about his experiment, what he
+learned, and what he would recommend to other designers who might want
+to test out the Draft Standards for their own design exercise.
+
+**Melody Kramer: How did you learn about the Draft Standards?**
+
+**Andrew Miller:** I was introduced to the design standards by colleagues
+here at Sandia, while looking for examples of pattern libraries and
+application frameworks. I was interested in finding that the Draft Web
+Standards were able to seamlessly blend accessibility with minimal and
+lightweight visual guidelines that could be scaled up and adapted to
+meet the specific needs of any application; I was hooked immediately.
+
+![An example of cia.gov redesigned with the Draft U.S. Web Design Standards]({{site.baseurl}}/assets/blog/web-design-standards/cia-redesign.jpg)
+*Andrew Miller used the Draft Standards pattern library to
+prototype a new cia.gov experience as a design exercise.*
+
+**MK: You decided to mock redesign the CIA website and are working on a mock redesign of the IRS? Why?**
+
+**AM:** It started when I was reading about applying the Draft Standards. I
+read articles introducing the Draft Standards and discussing how
+government websites were going to be attractive and intuitive soon. I
+went to the [Dashboard page](https://18f.gsa.gov/dashboard/) of 18F to
+start looking how other groups were using the Draft Standards and found
+a lot of great examples. I could really see the standards morph and
+adapt with each project, yet still retain the same principles, tone, and
+style.
+
+But I found that there wasn't an easy way to compare or begin to
+quantify the effect of following the standards. Some of the examples
+were initiatives that were new and didn't have an "old" version. So I
+wanted to apply the visual standards with a government household name,
+but one that wouldn't necessarily be on the shortlist for redesign. I
+wanted to excite people, and to show that ANY government website can
```

Comment:
I think we don't need this comma before the "and" because what follows isn't a complete sentence.